docker-compose/docker-compose.multiserver.mariadb.yml

```diff
@@ -24,11 +26,16 @@ services:
       - MYSQL_DATABASE=opencast
       - MYSQL_USER=opencast
       - MYSQL_PASSWORD=opencast
-    ports:
-      - "3306:3306"
```

Comment:
No strong feelings here. I published the port just for connecting to the DB locally. It is probably better to remove it as this should not be published in production.

Reply:
If you have the same port outside creates a possible problem if you have a MySQL running from before, the thing I can do is put another exit port or left without exit. ;)

Reply:
Again this is fine by me. Just wanting to explain the reason for publishing the port ;)